### PR TITLE
Add initial config for Vercel

### LINKED ISSUE #534: [IMPORTANT] Migrate from Zeit 1.0 config to Vercel config
This is essential to prevent the website from returning HTTP 500 errors.

Related to #495, #512, #529 and #531

// @maxbeatty @mathiasbynens

### PR DESCRIPTION
**Pull request by asilluron**
closes #534

### CODE REVIEW

**Review by mathiasbynens — APPROVED** [review]

File: vercel.prod.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "name": "JSPerf",
```
Comment (mathiasbynens) [inline vercel.prod.json:2]:
```suggestion
    "name": "jsPerf",
```

File: vercel.json [hunk 2]
```diff
@@ -1,12 +1,10 @@
 {
+  "name": "jsPerf",
+  "version": 2,
   "public": true,
-  "type": "docker",
-  "features": {
-    "cloud": "v2"
-  },
-  "alias": "jsperf.com",
+  "alias": ["jsperf.com"],
```
Comment (styfle) [inline vercel.json:5]: `alias` and `name` are deprecated, the can be removed from `vercel.json`

Instead, you should add the production domain to your Vercel Project in the dashboard.

File: vercel.prod.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+    "name": "jsPerf",
+    "version": 2,
+    "public": true,
+    "alias": ["jsperf.com"],
+    "env": {
+      "NODE_ENV": "production",
```
Comment (styfle) [inline vercel.prod.json:7]: You don't need another config file here if you're only changing env vars.

Instead, use the project settings to configure different values for Preview vs Production.

https://vercel.com/blog/environment-variables-ui

Furthermore, if you are only using `NODE_ENV`, for your serverless functions, then you shouldn't need to assign this at all.

**Comment by L3P3** [comment]
I need jsperf, how can I help to bring it back up quickly?

**Pull request merged**